libimage/load.go

```diff
@@ -35,6 +35,17 @@ func (r *Runtime) Load(ctx context.Context, path string, options *LoadOptions) (
 	var loadErrors []error
 
 	for _, f := range []func() ([]string, string, error){
+		// DOCKER-ARCHIVE - must be first (see containers/podman/issues/10809)
```

Comment:
This seems unrelated and is probably re-introducing a bug.

Reply:
Fixed now, I messed up my rebase initially